webpack.config.js

```diff
@@ -6,7 +6,8 @@ const CleanCSSPlugin = require("less-plugin-clean-css");
 module.exports = {
     entry: {
         demo: ["core-js/stable", "regenerator-runtime/runtime", path.join(__dirname, "src/js/demo/index.js")],
-        parser: ["core-js/stable", "regenerator-runtime/runtime", path.join(__dirname, "src/js/parser/index.js")]
+        parser: ["core-js/stable", "regenerator-runtime/runtime", path.join(__dirname, "src/js/parser/index.js")],
+        main: ["core-js/stable", "regenerator-runtime/runtime", path.join(__dirname, "src/js/main/index.js")]
```

Comment:
There must be a better way to bundle `core-js/stables` and `regenerator-runtime/runtime` than this. It's a little awkward because the demo/parser scripts are loaded first.

cc @hzoo Any ideas of how we could make this better (ideally a separate bundle that can be cached by the browser?).

Reply:
Going to merge this as is for now and explore this in a future PR.

Reply:
Related PR: https://github.com/eslint/eslint.github.io/pull/598